CMakeLists.txt

```diff
@@ -74,13 +74,37 @@ if(NOT PARAVIEW_QT_VERSION STREQUAL "5")
     "Tomviz requires PARAVIEW_QT_VERSION to be 5, please rebuild ParaView.")
 endif()
 
-find_package(ITK 4.9)
+find_package(ITK 4.9 REQUIRED)
 if(ITK_FOUND AND NOT ITK_WRAP_PYTHON)
   message(FATAL_ERROR
     "Tomviz requires ITK_WRAP_PYTHON to be enabled. "
     "Please rebuild ITK with ITK_WRAP_PYTHON set to TRUE.")
 endif()
 
+# Verify that the ITK build is using the same Python as the ParaView build
+load_cache(${ITK_DIR}
+  READ_WITH_PREFIX ITK_
+  PYTHON_INCLUDE_DIR PYTHON_LIBRARY
+)
+load_cache(${ParaView_DIR}
+  READ_WITH_PREFIX ParaView_
+  PYTHON_INCLUDE_DIR PYTHON_LIBRARY
+)
+
+if (NOT ITK_PYTHON_INCLUDE_DIR STREQUAL ParaView_PYTHON_INCLUDE_DIR)
+  message(FATAL_ERROR
+    "Python include directories are not the same for both ParaView and ITK."
```

Comment:
This needs a space at the end, or a newline so that the period doesn't run into the first word of the new sentence. I would also like to see it print out the two include directories for easier debugging, as shown in the failing CI builds.